### PR TITLE
Add data source Trino

### PR DESCRIPTION
**Pull request by grieve54706**
# Description
Make `POST /v2/connector/{data_source}/query` support new data source `trino`

## Request body
- host: string, `required`
- port: integer, `required`
- catalog: string, `required`
- schema: string, `required`
- user: string, `required`
- password: string
```json
{
  "sql": "select * from \"Orders\" limit 1",
  "manifestStr": "eyJjYXRhbG9nIjoibXlfY2F0YWxvZyIsInNjaGVtYSI6Im15X3...",
  "connectionInfo": {
    "host": "localhost",
    "port": 8080,
    "catalog": "catalog",
    "schema": "schema",
    "user": "username",
    "password": "password"
  }
}
```
or
- connectionUrl: string, `required`
```json
{
  "sql": "select * from \"Orders\" limit 1",
  "manifestStr": "eyJjYXRhbG9nIjoibXlfY2F0YWxvZyIsInNjaGVtYSI6Im15X3...",
  "connectionInfo": {
    "connectionUrl": "trino://username:password@localhost:8080/catalog/schema"
  }
}
```

## Response body
- columns
- data
- dtypes: Type through the pandas

```json
{
    "columns": [
        "o_orderkey",
        "o_custkey",
        "o_orderstatus",
        "o_totalprice",
        "o_orderdate",
        "o_orderpriority",
        "o_clerk",
        "o_shippriority",
        "o_comment"
    ],
    "data": [
        [
            1,
            370,
            "O",
            172799.49,
            820540800000,
            "5-LOW",
            "Clerk#000000951",
            0,
            "nstructions sleep furiously among "
        ]
    ],
    "dtypes": {
        "o_orderkey": "int32",
        "o_custkey": "int32",
        "o_orderstatus": "object",
        "o_totalprice": "object",
        "o_orderdate": "object",
        "o_orderpriority": "object",
        "o_clerk": "object",
        "o_shippriority": "int32",
        "o_comment": "object"
    }
}
```

# Additional information
Add [taplo](https://github.com/tamasfe/taplo) to format toml.
Testcontainers does not support Trino, so I created TrinoContainer for ourselves. I will add a PR to [testcontainers-python](https://github.com/testcontainers/testcontainers-python).

### CODE REVIEW

**Comment by grieve54706** [comment]
Follow https://github.com/testcontainers/testcontainers-python/issues/641

File: ibis-server/docker-compose.yml
Comment (goldmedal) [inline ibis-server/docker-compose.yml:1]: What's the file's purpose? example?

**Review by goldmedal — APPROVED** [review]
Thanks @grieve54706 overall looks good but some minor comments.

File: ibis-server/app/main.py [hunk 1]
```diff
@@ -23,6 +26,8 @@ def health():
 
 @app.get("/config")
 def config():
+    logger.info("/config")
+    logger.debug("THIS IS DEBUG")
```
[the remark this answers is not in the document]
Comment (grieve54706) [reply ibis-server/app/main.py:30]: I'm so sorry about it. It's for other tests. I thought I deleted it. I will add another commit to remove it.

File: ibis-server/docker-compose.yml
Comment (grieve54706) [reply ibis-server/docker-compose.yml:1]: I will remove it too.

File: ibis-server/justfile [hunk 2]
```diff
@@ -53,3 +53,4 @@ format:
 fmt:
     poetry run ruff format .
     poetry run ruff check --fix .
+    taplo fmt
```
[the remark this answers is not in the document]
Comment (grieve54706) [reply ibis-server/justfile:56]: I don't have any idea about changing the configuration of `taplo`. So let's put off it.

**Pull request merged**